hls-plugin-api/src/Ide/Types.hs

```diff
@@ -470,6 +477,9 @@ instance PluginRequestMethod TextDocumentPrepareCallHierarchy where
 instance PluginRequestMethod TextDocumentSelectionRange where
   combineResponses _ _ _ _ (x :| _) = x
 
+instance PluginRequestMethod TextDocumentFoldingRange where
+  combineResponses _ _ _ _ (x :| _) = x
```

Comment:
Should we instead concatenate the result, instead of always taking the first one?

Reply:
Yes, definitely.

Reply:
Yes! Makes sense